android/app/src/main/java/com/mattermost/rnbeta/MainApplication.java

```diff
@@ -197,8 +197,9 @@ public ReactModuleInfoProvider getReactModuleInfoProvider() {
             new SvgPackage(),
             new LinearGradientPackage(),
             new ReactVideoPackage(),
-            new RNGestureHandlerPackage()
+            new RNGestureHandlerPackage(),
 //            new RNReactNativeHapticFeedbackPackage()
```

Comment:
weird that this passed review, if you can perhaps as a favor you can remove this commented line?

Reply:
no problem, i have removed it